.gitignore

```diff
@@ -8,3 +8,4 @@ upload
 /.DS_Store
 /.idea
 /.vscode/settings.json
+/config/c4d-app-c299b-firebase-adminsdk-vvvu2-e95160bc8e.json
```

Comment:
We must not add this file to GitHub please delete it ASAP
please tell rami to upload it to the server manually